client/src/components/Typeahead/Typeahead.stories.js

```diff
@@ -109,3 +107,9 @@ BasicTypeahead.args = {
   min_length: 2,
   data,
 };
+
+export const HeaderTypeahead = TypeaheadTemplate.bind({});
+HeaderTypeahead.args = {
+  min_length: 2,
+  data,
+};
```

Comment:
Identical to the other typeahead story?

Reply:
Yeah, I had planned to do it later but I'll just delete it for now.